scripts/create_images.py

```diff
@@ -0,0 +1,153 @@
+"""
+ * @file create_image.py
+ * @brief Creates 2D images of TPs, plotting channel vs time. Colors represent the charge of the TP.
+ *
+ * This is part of the DUNE DAQ Application Framework, copyright 2024.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import time
+import os
+import argparse
+import warnings
+import gc
+
+import sys
+sys.path.append('../python/') 
+from utils import save_tps_array, create_channel_map_array
+from hdf5_converter import convert_tpstream_to_numpy 
+from image_creator import save_image, show_image
+from group_maker import make_groups
+
+
+# parser for the arguments
+parser = argparse.ArgumentParser(description='Tranforms Trigger Primitives to images.')
+parser.add_argument('-i', '--input-file',           type=str,                         help='Input file name')
+parser.add_argument('-o', '--output-path',          type=str,    default='./',        help='path to save the image')
+parser.add_argument('-n', '--n-tps',                type=int,    default=1,           help='number of tps to process')
+parser.add_argument('-t', '--ticks-limit',          type=int,    default=3,           help='closeness in ticks to group TPs')
+parser.add_argument('-c', '--channel-limit',        type=int,    default=1,           help='closeness in channels to group TPs')
+parser.add_argument('-m', '--min-tps',              type=int,    default=3,           help='minimum number of TPs to create a group and then an image')
+parser.add_argument('--which-detector',             type=str,    default="APA",       help='"APA", "CRP" or "50L"')
```

Comment:
I would call this channel map instead of "which-detector"

Reply:
Fixed in cc5e698